pytorch_lightning/trainer/trainer.py

```diff
@@ -1669,7 +1669,7 @@ def amp_backend(self) -> Optional[str]:
 
     @property
     def precision(self) -> Union[str, int]:
-        return self.accelerator.precision
+        return self.training_type_plugin.precision
```

Comment:
```suggestion
        return self.training_type_plugin.precision_plugin.precision
```
in case we want to skip the property definition as shared in my comment above.